bedrock/firefox/templates/firefox/hello/index.html

```diff
@@ -6,11 +6,11 @@
 
 {% extends "firefox/base-resp.html" %}
 
-{% set_lang_files "firefox/hello" %}
+{% set_lang_files "firefox/hello-2016" %}
```

Comment:
Just to be sure, I note that `index-b` uses `{% set_lang_files "firefox/hello/index-2016" %}` as apposed to `{% set_lang_files "firefox/hello-2016" %}`